scripts/entry

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
nit: should this also be `#!/usr/bin/env bash` for consistency with the other scripts?

Reply:
@superseb can you please check this ^, otherwise we should be good to merge.

Reply:
The difference here is that this is a Dapper entrypoint and not one of the scripts to be executed after entry. And going back to https://github.com/rancher/go-skel/blob/master/scripts/entry#L1 it apparently always was a static she-bang. I don't have a strong preference what to choose here, to keep it consistent with all the other repositories we should keep it. To make it consistent with the other scripts in this repository, we should change it. I guess we can keep it how it is unless we run into issues.